### PR TITLE
Added Firejail

### PR DESCRIPTION
**Pull request by Xmetalfanx**
Closes https://github.com/wimpysworld/deb-get/issues/234

tested and it seems to work 

(I'd have to lookup how to actually use this but commands from the official firejail documentation work after the install)

### CODE REVIEW

File: deb-get [hunk 1]
```diff
@@ -2413,6 +2413,17 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_firejail() {
+    get_github_releases "https://api.github.com/repos/netblue30/firejail/releases/latest"
+    if [ "${ACTION}" != "prettylist" ]; then
+        URL="$(grep "browser_download_url.*.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)"
```
Comment (natanjunges) [inline deb-get:2419]: Multiple files are provided, depending on the architecture. The correct file should be selected based on `HOST_ARCH`, and the supported architectures should be listed in `ARCHS_SUPPORTED`. Read [CONTRIBUTING#creating-the-function](https://github.com/wimpysworld/deb-get/blob/main/CONTRIBUTING.md#creating-the-function) for further information.

**Review by natanjunges — CHANGES_REQUESTED** [review]

File: deb-get [hunk 2]
```diff
@@ -2413,6 +2413,17 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_firejail() {
+    get_github_releases "https://api.github.com/repos/netblue30/firejail/releases/latest"
+    if [ "${ACTION}" != "prettylist" ]; then
+        URL="$(grep "browser_download_url.*${ARCHS_SUPPORTED}.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)"
```
Comment (natanjunges) [inline deb-get:2419]: Did you mean `HOST_ARCH` instead of `ARCHS_SUPPORTED`?

**Review by natanjunges — CHANGES_REQUESTED** [review]

**Review by flexiondotorg — CHANGES_REQUESTED** [review]
Please drop the Virtualbox change in README.md, rebase and squash the commits.

**Comment by natanjunges** [comment]
The format of package definitions has changed. Please read [01-main/CONTRIBUTING](https://github.com/wimpysworld/deb-get/blob/main/01-main/CONTRIBUTING.md) and [EXTREPO](https://github.com/wimpysworld/deb-get/blob/main/EXTREPO.md), update your implementation and submit a new PR.

**Pull request closed without merging**